docs/_components/search-bar.md

```diff
@@ -27,7 +27,7 @@ lead: A block that allows users to search for specific content if they know what
 
   <div class="usa-grid">
     <div class="usa-width-one-half">
-      <form class="usa-search">
+      <form class="usa-search usa-search-medium">
```

Comment:
The medium size is the default search size, so we do not want an additional class for it. Can you remove this and style `usa-search` directly?